DANS-KNAW/dd-verify-migration
# DD-711: Predict dataverse files from easy_files in frsdb (fedora)

Fixes DD-711: Predict dataverse files from easy_files in frsdb (fedora)

# Description of changes
 * core functionality in `EasyFileLoader`
 * see also javadoc of  getters at the end of `FedoraToBagCsv` and `EasyFileLoaderTest`

# How to test

* in the root of easy-dtap: `start-preprovisioned-box.py`
* in the root of dd-verify-file-migration: 
  * once: run `start-env.sh` and change the password for the easyDb in `etc/config.yml`
  * per test (each command line in another terminal, or the second command in Intellij by running DdVerifyFileMigrationApplication configured with the specified arguments):

        rm -rf data;start-hsqldb.sh
        start.sh load-from-fedora etc/config.yml src/test/resources/deasy.csv does-not-exist.csv

Or build, deploy on deasy, copy `src/test/resources/deasy.csv` to `easy-dtap/shared` and:

    dd-verify-file-migration load-from-fedora /etc/opt/dans.knaw.nl/dd-verify-file-migration/config.yml /vagrant/shared/deasy.csv

# Related PRs

* deploy with https://github.com/DANS-KNAW/easy-dtap/pull/528

# Notify
@DANS-KNAW/dataversedans
